docs/new-project-setup.md

```diff
@@ -0,0 +1,310 @@
+and after fifteen minutes of inactivity for particularly sensitive accounts.
+
+!!! note
+    It's possible we will do this using Terraform in the future,
+    but at the time of this writing the Snowflake Terraform provider does not support session policies.
+
+After the Snowflake account is created, run the following script in a worksheet
+to set the appropriate session policy:
+
+```sql
+use role sysadmin;
+create database if not exists policies;
+create session policy if not exists policies.public.account_session_policy
+  session_idle_timeout_mins = 60
+  session_ui_idle_timeout_mins = 60
+;
+use role accountadmin;
+-- alter account unset session policy;  -- unset any previously existing session policy
+alter account set session policy policies.public.account_session_policy;
+```
+
+### Add IT-Ops representatives
+
+TODO: establish and document processes here.
+
+### Set up Okta SSO and SCIM
+
+TODO: establish and document processes here.
+
+## Create project git repository
+
+Create a new git repository from the CalData Infrastructure Template
+following the instructions [here](https://github.com/cagov/caldata-infrastructure-template#usage).
+
+Once you have created the repository, push it to a remote repository in GitHub.
+There are some GitHub actions that will fail because the repository is not yet
+configured to work with the new Snowflake account.
+
+## Deploy project infrastructure using Terraform
+
+We will create two separate deployments of the project infrastructure,
+one for development, and one for production.
+In some places we will refer to project name and owner as `<project>` and `<owner>`, respectively,
+following our [naming conventions](./naming-conventions.md).
+You should substitute the appropriate names there.
+
+### Create the dev configuration
+
+1. In the new git repository, create a directory to hold the development Terraform configuration:
+    ```bash
+    mkdir -p terraform/environments/dev/
+    ```
+    The location of this directory is by convention, and subject to change.
+1. Copy the terraform configuration from
+    [here](https://github.com/cagov/data-infrastructure/blob/main/terraform/snowflake/environments/dev/main.tf)
+    to your `dev` directory.
+1. In the "elt" module of `main.tf`, change the `source` parameter to point to
+    `"github.com/cagov/data-infrastructure.git//terraform/snowflake/modules/elt?ref=<ref>"`
+    where `<ref>` is the short hash of the most recent commit in the `data-infrastructure` repository.
+1. In the `dev` directory, create a new backend configuration file called `<owner>-<project>-dev.tfbackend`.
+    The file will point to the S3 bucket in which we are storing terraform state:
+    ```hcl
+    bucket = "dse-snowflake-dev-terraform-state"
+    dynamodb_table = "dse-snowflake-dev-terraform-state-lock"
+    key = "<owner>-<project>-dev.tfstate"
+    region = "us-west-2"
+    ```
+1. In the `dev` directory, create a terraform variables file called `terraform.tfvars`,
```

Comment:
are we meant to populate the new file with just the two lines of code listed below? or is this info about "elt" module variables that should also be there? @ian-r-rose